src/style.css

```diff
@@ -55,18 +57,176 @@ header,
-  font-size: 1rem;
-  margin: 20px 0;
-  border: 3px solid;
-  height: 50px;
-  width: 20vw;
+.form {
+  position: relative;
+  width: 20rem;
+  height: 3rem;
+}
+
+.form__input {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border: 2.5px solid rgb(10, 109, 149);
+  border-radius: 8px;
+  font-family: inherit;
+  font-size: 19px;
+  color: rgb(14, 13, 13);
+  outline: none;
+  padding: 1.25rem;
+  background: none;
+}
+
+.form__input:hover {
+  border-color: rgb(142, 212, 213);
+}
+
+.form__input:focus {
+  border-color: rgb(8, 174, 239);
+}
+
+.form__label {
+  position: absolute;
+  left: 1rem;
+  top: 0.8rem;
+  padding: 0 0.5rem;
+  color: rgb(29, 48, 66);
+  cursor: text;
+  transition: top 200ms ease-in, left 200ms ease-in, font-size 200ms ease-in;
+}
+
+.form__input:focus ~ .form__label,
+.form__input:not(:placeholder-shown).form__input:not(:focus) ~ .form__label {
+  top: -0.4rem;
+  font-size: 0.8rem;
+  left: 0.8rem;
+  background-color: #fff;
+}
+
+.form__input:not(:-moz-placeholder-shown).form__input:not(:focus) ~ .form__label {
+  top: -0.4rem;
+  font-size: 0.8rem;
+  left: 0.8rem;
+  background-color: #fff;
+}
+
+.form__input:not(:-ms-input-placeholder).form__input:not(:focus) ~ .form__label {
+  top: -0.4rem;
+  font-size: 0.8rem;
+  left: 0.8rem;
+  background-color: #fff;
+}
+
+.form2 {
+  position: relative;
+  width: 20rem;
+  height: 3rem;
+  margin-top: 20px;
+}
+
+.form2__input2 {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border: 2.5px solid rgb(10, 109, 149);
+  border-radius: 8px;
+  font-family: inherit;
+  font-size: 19px;
+  color: rgb(14, 13, 13);
+  outline: none;
+  padding: 1.25rem;
+  background: none;
+}
+
+.form2__input2:hover {
+  border-color: rgb(142, 212, 213);
+}
+
+.form2__input2:focus {
+  border-color: rgb(8, 174, 239);
+}
+
+.form2__label {
+  position: absolute;
+  left: 1rem;
+  top: 0.8rem;
+  padding: 0 0.5rem;
+  color: rgb(29, 48, 66);
+  cursor: text;
+  transition: top 200ms ease-in, left 200ms ease-in, font-size 200ms ease-in;
+}
+
+.eye {
+  position: absolute;
+  margin-left: 280px;
+  padding-top: 12px;
+}
+
+.fa {
+  font-size: 20px;
+  color: #7a797e;
+}
+
+.form2__input2:focus ~ .form2__label,
+.form2__input2:not(:placeholder-shown).form2__input2:not(:focus) ~ .form2__label {
+  top: -0.4rem;
+  font-size: 0.8rem;
+  left: 0.8rem;
+  background-color: #fff;
+}
+
+.form2__input2:not(:-moz-placeholder-shown).form2__input2:not(:focus) ~ .form2__label {
+  top: -0.4rem;
+  font-size: 0.8rem;
+  left: 0.8rem;
+  background-color: #fff;
+}
+
+.form2__input2:not(:-ms-input-placeholder).form2__input2:not(:focus) ~ .form2__label {
+  top: -0.4rem;
+  font-size: 0.8rem;
+  left: 0.8rem;
+  background-color: #fff;
+}
+
+.form-container {
+  padding: 30px 0;
+}
+
+.Coll-2 {
+  padding-left: 20px;
+}
+
+@media only screen and (max-width: 890px) {
+  .Content {
+    flex-direction: column;
+    margin: 5%;
+  }
+
+  .score {
```

Comment:
- [optional] You can also work more on the styling for the mobile version, I could noticed that the elements are overlapped. This is not a good UX, try and adjust. 👍

Reply:
alright sure